doc/source/serve/index.rst

```diff
@@ -176,6 +176,51 @@ Note that the count has been reset to zero because the new version of ``Counter`
   > curl -X GET localhost:8000/Counter/decr
   {"count": 0}
 
+Serve also lets you set configurations in the ``@serve.deployment`` decorator
+or in a YAML config file. For example, you can set the
+``max_concurrent_queries`` for your application:
+
+.. code-block:: python
+
+  # File name: counter_app.py
+
+  @serve.deployment(
+    max_concurrent_queries=10
+  )
+  @serve.ingress(app)
+  class Counter:
+  ...
+
+  Counter.deploy()
+
+Instead of modifying the decorator, you can write a YAML config file:
+
+.. code-block:: yaml
+  
+  # File name: counter_config.yaml
+
+  deployments:
+    
+    - name: Counter
+      import_path: counter_app.Counter
+      max_concurrent_queries: 10
+
+Then you can use the Serve CLI, which is included with Ray Serve, to update and
+deploy ``Counter``:
+
+.. code-block:: bash
+  
+  $ serve run counter_config.yaml
```

Comment:
it seems more natural to add `serve run` examples in the pipelines documentation that we will add; here it's just fragmenting the UX for now clear benefit

let's discuss at a higher level what we want to document and where before proceeding (in the doc you're working on)